src/main/java/ezvcard/io/json/JCardWriter.java

```diff
@@ -1,224 +1,286 @@
-package ezvcard.io.json;
```

Comment:
Kind of annoying that the diff thinks the entire `JCardWriter.java` file was changed.  Do you know why?